### DISCUSSION TITLE
Custom "link" in a statblock

### DISCUSSION THREAD

**Discussion by alphacentor31** [Obsidian-TTRPG-Community/fantasy-statblocks#145] [Q&A]
First of all thank you very much for your work! I'm a french guy (and sorry for my poor english) who was looking for an obsidian pluggin like this and TTrpg statblock is amazing.

I organize RPGs using the "Talespire" virtual table top and based on the "savage worlds" game system. In talespire with the help of an internet browser you can easily type strings of characters allowing to have easy roll of dice on your virtual board (like this for example: talespire://dice/2d12). Very handy for the GM.

So  I would like to integrate this into my files created with your plugin but I have no idea how to do it in a simple way. Because my first idea was to use a markdown (ex : "[Roll_force](talespire://dice/1d12)") in a text block of the layout... But that seems to crash the statblock (probably I'm stupid, is a text block !). Is there a layout element type to make a personal link functional?

**Comment by sigrunixia** [comment]
Fantasy Statblock cannot work with external links.
